pegasus/sites.v3/code.org/views/dance_tutorial_section.haml

```diff
@@ -1,3 +1,5 @@
+- hide_dance_extra_button = DCDO.get('dance_extra_hoc', nil)
```

Comment:
We use DCDO variables when we are pretty sure we'll need to change behavior on the site immediately without doing a DTP.  It would be good to work with a PM to find out whether this is a likely scenario here.

Reply:
Also, if we want this page to get eyes coverage on /test, we'll need a way to set this DCDO flag for the test environment.  See https://github.com/code-dot-org/code-dot-org/pull/25809 for an example of how to do that.